src/main/java/net/sf/jabref/logic/journals/AbbreviationParser.java

```diff
@@ -30,9 +30,12 @@ public void readJournalListFromResource(String resourceFileName) {
     }
 
     public void readJournalListFromFile(File file) throws FileNotFoundException {
-        readJournalList(new FileReader(Objects.requireNonNull(file)));
+        try(FileReader fr = new FileReader(Objects.requireNonNull(file))) {
+            readJournalList(fr);
+        } catch(IOException e) {
+            LOGGER.info("Could not read journal list from file " + file.getAbsolutePath());
+        }
```

Comment:
Yes, of course! Sort of obvious in hindsight.

Thanks for reviewing code and helpful suggestions!